### PR TITLE
feat(replays): new replays table column

### LINKED ISSUE #37070: Create an ambiguous column 'interest icon' with a BS calculation that takes pages viewed + time + errors and chooses an icon to show

### PR DESCRIPTION
**Pull request by jesus4497**
Added a new column for the replays table at the index page, showing an icon of interest. Closes #37070 

image

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/components/replays/replayHighlight.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import CHART_PALETTE from 'sentry/constants/chartPalette';
+import {ReplayDurationAndErrors} from 'sentry/views/replays/types';
+
+import ScoreBar from '../scoreBar';
```
Comment (ryan953) [inline static/app/components/replays/replayHighlight.tsx:6]: lets use absolute impors from now on, no more `./` or `../`

File: static/app/components/replays/replayHighlight.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const {count_if_event_type_equals_error: errors, 'equation[0]': durationInSeconds} =
+      data;
+
+    const pagesVisitedOverTime = pagesVisited / (durationInSeconds || 1);
+
+    score = (errors * 25 + pagesVisited * 5 + pagesVisitedOverTime) / 10;
+    score = Math.floor(Math.min(10, Math.max(1, score)));
+  }
+
+  const palette = new Array(10).fill([CHART_PALETTE[0][0]]);
```
Comment (ryan953) [inline static/app/components/replays/replayHighlight.tsx:27]: is this color going to work in dark mode?

Comment (jesus4497) [reply static/app/components/replays/replayHighlight.tsx:26]: Yes! 
image

**Review by ryan953 — APPROVED** [review]

**Pull request merged**